src/Containers/Containers.jl

```diff
@@ -26,13 +26,16 @@ export DenseAxisArray, SparseAxisArray
 include("DenseAxisArray.jl")
 include("SparseAxisArray.jl")
 
-Base.eachindex(g::Base.Generator{<:Union{DenseAxisArray, SparseAxisArray}}) = eachindex(g.iter)
+Base.eachindex(g::Base.Generator{<:Union{DenseAxisArray,SparseAxisArray}}) =
+    eachindex(g.iter)
```

Comment:
This is not allowed by our style guide, we should use `function` if ot is multiline

Reply:
The formatter only provides an option to re-write every single-line function, not an option to rewrite only those that are longer than the line limit.

Merging for now, and we can fix any offending examples by hand.

Reply:
I was wrong: https://github.com/jump-dev/JuMP.jl/pull/2385